temba/flows/models.py

```diff
@@ -2198,6 +2213,35 @@ def _get_run_batches(self, flows, responded_only):
         runs = FlowRun.objects.filter(flow__in=flows).order_by("modified_on")
         if responded_only:
             runs = runs.filter(responded=True)
+
+        # filter runs by extra queries
+        extra_queries = getattr(self, "extra_queries", {})
+        if extra_queries.get("response"):
+            response_queries = extra_queries.get("response", {})
+            if response_queries.get("after"):
+                after = self.org.parse_datetime(response_queries.get("after"))
+                runs = runs.filter(created_on__gte=after)
```

Comment:
Per Rob recommendation, we should change from `created_on` to `modified_on` for the before/after filter